addons/pylons/functions/fnc_onButtonSave.sqf

```diff
@@ -0,0 +1,36 @@
+/*
+ * Author: 654wak654
+ * Saves the selected pylon configuration to profileNamespace.
+ *
+ * Arguments:
+ * None
+ *
+ * Return Value:
+ * None
+ *
+ * Example:
+ * [] call ace_pylons_fnc_onButtonSave
+ *
+ * Public: No
+ */
+#include "script_component.hpp"
+
+private _loadoutName = ctrlText ID_EDIT_LOADOUTNAME;
+private _aircraftLoadouts = profileNamespace getVariable [QGVAR(aircraftLoadouts), []];
+private _loadoutPylons = GVAR(comboBoxes) apply {(_x select 0) lbData (lbCurSel (_x select 0))};
+
+private _found = {
```

Comment:
save wasn't working
_found is nil
might works differently for you because _aircraftLoadouts isn't []